pkg/apis/sedna/v1alpha1/jointinferenceservice_types.go

```diff
@@ -40,6 +41,7 @@ type JointInferenceService struct {
 type JointInferenceServiceSpec struct {
 	EdgeWorker  EdgeWorker  `json:"edgeWorker"`
 	CloudWorker CloudWorker `json:"cloudWorker"`
+	appsv1.DeploymentSpec `json:",inline"`
```

Comment:
Why add a definition of DeploymentSpec to the api

Reply:
It's my oversight. The definition was not used in the code and I have removed it.